aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1176 @@
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+{
+  grafanaDashboards+:: {
+    'aerospike-overview.json':
+      dashboard.new(
+        'Aerospike overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+      .addLink(grafana.link.dashboards(
+        asDropdown=false,
+        title='Other Aerospike Dashboards',
+        includeVars=true,
+        keepTime=true,
+        tags=($._config.dashboardTags),
+      ))
+      .addTemplates(
+        [
+          template.datasource(
+            promDatasourceName,
+            'prometheus',
+            null,
+            label='Data Source',
+            refresh='load'
+          ),
+          template.new(
+            'job',
+            promDatasource,
+            'label_values(aerospike_namespace_ns_cluster_size,job)',
+            label='Job',
+            refresh=2,
+            includeAll=false,
+            multi=false,
+            allValues='',
+            sort=0
+          ),
+          template.new(
+            'cluster',
+            promDatasource,
+            'label_values(aerospike_node_stats_system_free_mem_pct{%(multiclusterSelector)s}, cluster)' % $._config,
+            label='Cluster',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='',
+            hide=if $._config.enableMultiCluster then '' else 'variable',
+            sort=0
+          ),
+          template.new(
+            'aerospike_cluster',
+            promDatasource,
+            'label_values(aerospike_namespace_ns_cluster_size{%(aerospikeSelector)s},aerospike_cluster)' % $._config,
+            label='Aerospike cluster',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='',
+            sort=0
+          ),
+          template.new(
+            'k',
+            promDatasource,
+            '5, 10, 20, 50',
+            label='K',
```

Comment:
@Caleb-Hurshman Just commenting that it seems like we should be able to have a better label name than "K"? like "Top percentile" or whatever? What do you think?

Reply:
We're actually not using a percentile anymore. I think renaming it would be good as 'k' isn't super clear.

Reply:
@Caleb-Hurshman I still think "Top K" might be unclear for some customers. How about something more along the lines of "Top node count" or something like that?

Reply:
To be clear I'm referring to the dashboard where this selector still exists

Reply:
Will change, but not sure what you mean by your clarifying comment as this selector has only ever existed on one dashboard.